### ISSUE TITLE
Figure Out How Many Stars to Pick for Galaxy

### ISSUE THREAD

**Issue by robelgeda** [robelgeda/dsii_dwarfs#8]

**Comment by hcferguson** [comment]
This is implemented in HSC_dwarf_simulation. We compute the mass fraction in the stochastic component in cell 6:

```
kroupa = imf.Kroupa(mmin=0.1, mmax=120, p1=0.3, p2=1.3, p3=2.3,
                 break1=0.08, break2=0.5)
kroupa.normalize()
mmax = 200
stochastic_mass_fraction = (kroupa.m_integrate(minmass_stochastic, mmax)[0] /
                 kroupa.m_integrate(kroupa.mmin, mmax)[0])
stochastic_mass_fraction
```

And then make a cluster of this mass in cell 14:

```
cluster = imf.make_cluster(stochastic_mass_fraction*galaxy.total_mass*catalog_extra,
                           massfunc='kroupa',mmin=minmass_stochastic)
```

**Comment by hcferguson** [comment]
Closed by #17.
